dash_docs/chapters/testing/index.py

```diff
@@ -44,22 +44,31 @@
 
     FYI, We run Dash integration tests with Chrome WebDriver.
     But the fixture allows you to choose another browser from the command line,
-    e.g. `pytest --webdriver Firefox -k bsly001`.
+    e.g. `pytest --webdriver Firefox -k <test directory>/<test name>`.
 
-    Headless mode is added in Dash *1.0.1*, run `pytest --headless -k bsly001`
+    Headless mode is added in Dash *1.0.1*, run `pytest --headless -k <test directory>/<test name>`
     to start test in headless mode. First time hearing about `headless mode`? The
     main benefit for us is it's lighter and faster to run without a UI. You
     can check the details from both [Firefox](https://developer.mozilla.org/en-US/docs/Mozilla/Firefox/Headless_mode)
     and [Chrome](https://developers.google.com/web/updates/2017/04/headless-chrome).
 
     Remote WebDriver support is added in Dash *1.3.0*. There are two ways to use it:
 
-    1. Run `pytest --remote -k bsly001` to grab a Chrome WebDriver from a local
+    1. Run `pytest --remote -k <test directory>/<test name>` to grab a Chrome WebDriver from a local
     hosted grid at `http://localhost:4444/wd/hub`
     2. Run `pytest --webdriver Firefox --remote-url https://grid_provioder_endpoints`
     to connect with a remote grid in the cloud running Firefox (default Chrome).
     Note that you don't need to use `--remote` as soon as the `--remote-url`
     value is set and different than the default one.
+    * You can use `--remote-url http://localhost:4444/wd/hub` when you are using standlone version of 
+    chrome or firefox.
+    * You can start the stand alone docker containers for firefox:
+    `docker run -d -p 4444:4444 -p 7900:7900 --shm-size 2g selenium/standalone-firefox`
+    * You can start the stand alone docker container for chrome with:
+    `docker run -d -p 4444:4444 -p 7900:7900 --shm-size 2g selenium/standalone-chrome`
+    * Documentation for these docker containers is here: [docker-selenium](https://github.com/SeleniumHQ/docker-selenium)
```

Comment:
Thanks for the extra detail here!

```suggestion
    * You can use `--remote-url http://localhost:4444/wd/hub` when you are using standalone version of 
    Chrome or Firefox.
    * To start a standalone docker container for Firefox:
    `docker run -d -p 4444:4444 -p 7900:7900 --shm-size 2g selenium/standalone-firefox`
    * To start a standalone docker container for Chrome:
    `docker run -d -p 4444:4444 -p 7900:7900 --shm-size 2g selenium/standalone-chrome`
    * These containers are documented at [docker-selenium](https://github.com/SeleniumHQ/docker-selenium).
```